Author: Jaromír Mikeš <mira.mikes@seznam.cz>
Description: Pass LDFLAGS
Forwarded: no

Index: fil-plugins/Makefile
===================================================================
--- fil-plugins.orig/Makefile
+++ fil-plugins/Makefile
@@ -23,7 +23,7 @@ all:	filters.so
 
 
 filters.so:	filters.o filters_if.o exp2ap.o
-	g++ -shared filters.o filters_if.o exp2ap.o -o filters.so
+	g++ $(LDFLAGS) -shared filters.o filters_if.o exp2ap.o -o filters.so
 
 filters.o:	ladspaplugin.h filters.h
 filters_if.o:	ladspaplugin.h filters.h
